src/pages/index.astro

```diff
@@ -0,0 +1,132 @@
+---
+import Layout from '../layouts/Layout.astro';
+---
+<Layout currentPage="Home">
+    	<main>
+		<div class='info'>
+			<div class='about'>
+				<h2>About</h2>
+				<p>
+					Redstone Tools is a <abbr title='Quality Of Life'>QOL</abbr> mod designed
+					to make building redstone contraptions quicker and easier by letting you
+					focus on the fun parts and letting us do the boring bits.
+				</p>
+			</div>
+			<div class='open-source'>
+				<h2>Open Source</h2>
+				<p>
+					The mod and everything it uses is open source meaning you can see and
+					contribute to the code, you can find the source code and information
+					on contributing at <a
+						href='https://github.com/redstonetools/redstonetools'
+						target='_blank'>GitHub</a
+					>.
+				</p>
+			</div>
+		</div>
+		<ul class='top-features'>
+			<li class='feature macros'>
+				<div>
+					<h3>Macros</h3>
+					<p>
+						Do you find yourself running the same commands over and over? Well
+						the macro system has you covered! configure a list of commands which
+						can be activated either with a command or a keybind.
+					</p>
+				</div>
+			</li>
+			<li class='feature worldedit-extensions'>
+				<div>
+					<h3>WorldEdit Extensions</h3>
+					<p>
+						Commands that extend worldedit such as //minsel which removes all
```

Comment:
I don't think we should mention any specific commands to prevent needing to change this